packages/networkaccess/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import { OnRpcRequestHandler } from '@metamask/snaps-types';
+import { panel, text } from '@metamask/snaps-ui';
+
+// eslint-disable-next-line jsdoc/require-jsdoc
```

Comment:
This eslint requirement is easy to fix, we should avoid using disable if it's not really needed.

Something like:
```
/**
 * Retrieves test web page content.
 *
 * @returns Response data of a fetched web page in text format.
 */
```

Reply:
resolved in latest commit

Reply:
It isn't fixed in this PR, did you push the changes?

Reply:
it should be pushed now...